src/client/components/ProfileFormComponent/ProfileFormComponent.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Avatar from './ProfileFormComponent';
+import PropTypes from 'prop-types';
+
+export default {
+  title: 'Avatar',
+  component: Avatar,
+  argTypes: {
+    color: {
+      control: { type: 'color' },
+    },
+    width: {
+      control: { type: 'range', min: 400, max: 1200, step: 50 },
+    },
```

Comment:
Don't we need one more control for fonts?

Reply:
yes tithi in the docs we have fonts also but when iam using in storybook i can't see any difference.maybe i will check that

Reply:
yes tithi we need fonts control also i did that to.